packages/plugin-core/src/common/EngineNoteProvider.ts

```diff
@@ -1,3 +1,4 @@
+import "reflect-metadata";
```

Comment:
was this one needed for a test to work? 

If so, I think we can configure this in Jest (lemme dig up some code for you)

Reply:
Na, this import isn't required. I might have added accidently.